### PR TITLE
Use the new Brackets event dispatcher

### PR DESCRIPTION
**Pull request by MarcelGerber**
... to avoid deprecation warnings in the console.

### CODE REVIEW

**Comment by zaggino** [comment]
thanks :+1:

File: src/Branch.js [hunk 1]
```diff
@@ -290,7 +290,7 @@ define(function (require, exports) {
     function attachCloseEvents() {
         $("html").on("click", closeDropdown);
         $("#project-files-container").on("scroll", closeDropdown);
-        $(SidebarView).on("hide", closeDropdown);
+        SidebarView.on("hide", closeDropdown);
```
Comment (zaggino) [inline src/Branch.js]: @MarcelGerber I'm getting now
```
Possibly unhandled TypeError: undefined is not a function
    at attachCloseEvents (file:///C:/Users/Zaggi/AppData/Roaming/Brackets/extensions/user/zaggino.brackets-git/src/Branch.js:293:21)
```

Comment (MarcelGerber) [inline src/Branch.js]: Well, `SideBarView` stopped triggering `hide` 2 years ago... https://github.com/adobe/brackets/commit/df7a4956c891baf5b41877fe7d18cbb4b69fba57#diff-ea2ced07a9500d942aeaa1f8844cf134L132

Comment (MarcelGerber) [inline src/Branch.js]: See also: Commit message of adobe/brackets@cb0b0231cd3d841e8eb8d6693abbb2f9171c5913

**Pull request merged**